### DISCUSSION TITLE
Option to Save As Source File Reference Only

### DISCUSSION THREAD

**Discussion by MusoCity** [audacity/audacity#5562] [Ideas & Feature Requests]
You are using non destructive editing now, you should have an option to save just as a reference file (like .xml) to just reference the file locations and not save a massive big *.aup3 file that includes all the audio data.
It would be good to have an **option**. I use a 2TB library of recorded session musicians in wma and wav format, it starts using too much hard drive space if you render all the files to new wav files unnecessarily. I can script Reaper to load all the source file sections used in Band In A Box as wma or wav (or flac aiff mp3 etc) without having to render tracks down to wavs on the hard drive then import them. Reaper plays all the formats direct without decompressing to wav first, so no new files are created.
https://www.reatrak.com/downloads/biab/Reaper-WMA-Sections.png

**Comment by LWinterberg** [comment]
A new file format absolutely is coming in the future and being able to do that would be a requirement from my side. It'll be a while until that happens though, we have an Audacity 4 to make first.

**Comment by MusoCity** [comment]
That sounds good. See the option in Reaper: https://www.reatrak.com/downloads/biab/reaper-save-options.png
